conditional/blueprints/member_management.py

```diff
@@ -603,6 +630,32 @@ def clear_active_members():
     return jsonify({"success": True}), 200
 
 
+@member_management_bp.route('/manage/export_active_list', methods=['GET'])
+def export_active_list():
+    sio = io.StringIO()
+    csvw = csv.writer(sio)
+
+    active_list = [["Full Name", "RIT Username", "Amount to Charge"]]
+    for member in ldap_get_active_members():
+        full_name = member.cn
+        rit_username = re.search(".*uid=(\\w*)", member.ritDn).group(1)
+        will_coop = CurrentCoops.query.filter(
+            CurrentCoops.date_created > start_of_year(),
+            CurrentCoops.semester != "Neither",
+            CurrentCoops.uid == member.uid).first()
+        if will_coop:
+            dues = 80
```

Comment:
This is probably 5-10 minutes of work, I don't see why it shouldn't be included. You just have to add an option in the config and then import the `app.config` into this file.